app/controllers/comments_controllers.rb

```diff
@@ -14,13 +14,13 @@ def show
   end
 
   def update
-    @comment =
-    @comment =
+    @comment = Comment.find(params[:id])
+    @comment.update(comment_params)
```

Comment:
Looking good.  Go find the requested comment.  Then update that comment.  "comment_params" should be a method that provides the "strong params" for the comment - it manages/identifies what params that can be passed to the `@comment.update` method.  You need to create the "comment_params" methods for this controller.